src/ctapipe/monitoring/tests/test_interpolator.py

```diff
@@ -57,7 +173,7 @@ def test_invalid_input():
     wrong_unit = Table(
         {
             "time": Time(1.7e9 + np.arange(3), format="unix"),
-            "azimuth": [1, 2, 3] * u.deg,
+            "azimuth": np.radians([1, 2, 3]) * u.rad,
```

Comment:
Why did you change these tests here?

Reply:
The test is explicitly to test the unit conversion, so please don't change to rad

Reply:
The pointing_interpolator is supposed to use radians for input. Line:
```
157         expected_units = {"azimuth": u.rad, "altitude": u.rad}
```

Reply:
right, and this test checks that correct exception is raised in case the units are not matching. Please revert back